# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/mysqlcc/ssl_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mysqlcc.pro.in.orig	2003-08-06 00:31:41.000000000 +0200
+++ ./mysqlcc.pro.in	2003-08-06 00:32:16.000000000 +0200
@@ -25,7 +25,7 @@
 }
 
 unix {
-      LIBS        += @MYSQL_LIB@ 
+      LIBS        += @MYSQL_LIB@ -lssl
       INCLUDEPATH += @MYSQL_INCLUDE@
       CONFIG      += x11 
       OBJECTS_DIR  = tmp
